ZenPacks/zenoss/ZenPackLib/zenpacklib.py

```diff
@@ -60,14 +61,108 @@
     Service,
     IpService,
     WinService
-    )
+)
 
 from lib.spec import ZenPackSpec
 from lib.functions import relationships_from_yuml, catalog_search, ucfirst, relname_from_classname
 
+LOG = logging.getLogger('zen.ZenPackLib')
+
+
 def enableTesting():
-    """deprecated"""
-    pass
+    """This is for backwards compatibility.  For future tests, use ZPLTestHarness
```

Comment:
I don't think this is really for backwards compatibility, as I understand it- enableTesting/TestCase is for regular unit tests in zenpacks.    ZPLTestHarness is for testing ZPL, right?